src/indicators/loading-container.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+/**
+ *
+ * A wrapper component that visually indicates whether a child component
+ * is loading, or loaded.
+ *
+ * 
+ * @name LoadingContainer
+ * @type Function
+ * @example
+ * 
+ * function PatientIndex ({ patientProfiles=true, isLoading=true }) {
```

Comment:
You can just use `patientProfiles` here and make `isLoading={ !patientProfiles }`

Reply:
Instead of the external `&&` condition